tests-node/unit/utils/strip-nested-nulls-test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env node */
+/* globals describe, it */
+
+'use strict';
+
+let expect = require('chai').expect;
+
+let stripNestedNulls = require('../../../lib/utils/strip-nested-nulls');
+
+describe('stripNestedNulls', function() {
+  [
+    [{ a: true, b: null }, { a: true }],
```

Comment:
Can you add a test ensure empty strings do not get stripped out

Reply:
done